docs/source/tools/mock_trafficops.rst

```diff
@@ -0,0 +1,99 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _mock-to:
+
+***************************
+The Mock Traffic Ops Server
+***************************
+.. versionadded:: 4.0.0
```

Comment:
Still on v3, probably 3.1 at this point

Reply:
So, I've been told conflicting things about this. Some people have told me that `master` is set to track the next major rev, some say the next minor.

Reply:
Sure, ask there. I'll chime in on that thread

Reply:
done